k8s-openapi-codegen-common/src/templates/impl_deserialize.rs

```diff
@@ -133,16 +133,14 @@ pub(crate) fn generate(
         writeln!(str_to_field_match_arms, "                            v => Field::Other(v.to_owned()),")?;
 
         writeln!(field_value_match_arms,
-            "                        Field::Other(key) => {{ value_{field_name}.insert({local}serde_value::Value::String(key), {local}serde::de::MapAccess::next_value(&mut map)?); }},")?;
+            "                        Field::Other(key) => {{ value_{field_name}.insert(key, {local}serde::de::MapAccess::next_value(&mut map)?); }},")?;
 
         writeln!(field_value_assignment,
             "                    {field_name}: {{")?;
         writeln!(field_value_assignment,
-            "                        let value_{field_name} = {local}serde_value::Value::Map(value_{field_name});")?;
+            "                        let value_{field_name} = {local}serde_json::Value::Object(value_{field_name});")?;
```

Comment:
`serde_json::Map` itself [is already `Deserializer`](https://docs.rs/serde_json/latest/serde_json/struct.Map.html#impl-Deserializer%3C'de%3E-for-Map%3CString,+Value%3E) so you should be able to skip wrapping it in `Value::Object`

Reply:
Thank you!